src/clarity.rs

```diff
@@ -203,21 +195,16 @@ fn get_cli_block_height(conn: &Connection, block_id: &StacksBlockId) -> Option<u
     );
     let mut rows = friendly_expect(stmt.query(&[block_id]), "FATAL: could not fetch rows");
     let mut row_opt = None;
-    while let Some(row_res) = rows.next() {
-        match row_res {
-            Ok(row) => {
-                let rowid = friendly_expect(
-                    u64::from_column(&row, "id"),
-                    "FATAL: could not parse row ID",
-                );
-                row_opt = Some(rowid);
-                break;
-            }
-            Err(e) => {
-                panic!("FATAL: could not read block hash: {:?}", e);
-            }
-        }
+
+    while let Some(row) = rows.next().expect("FATAL: could not read block hash") {
```

Comment:
The sqlite error message is no longer included, were they helpful?
`panic!("FATAL: could not read block hash: {:?}", e);`